demisto_sdk/scripts/changelog/tests/changelog_test.py

```diff
@@ -0,0 +1,160 @@
+from pathlib import Path
+
+import pytest
+
+from demisto_sdk.commands.common.handlers import YAML_Handler
+from demisto_sdk.scripts.changelog import changelog
+from demisto_sdk.scripts.changelog.changelog import (
+    Changelog,
+    clear_changelogs_folder,
+    get_all_logs,
+    get_new_log_entries,
+    is_log_folder_empty,
+    is_log_yml_exist,
+    is_release,
+)
+from demisto_sdk.scripts.changelog.changelog_obj import LogType
+
+yaml = YAML_Handler()
+
+
+LOG_FILE_1 = {
+        "logs": [{"description": "fixed an issue where test", "type": "fix"}],
+        "pr_number": "12345",
+    }
+LOG_FILE_2 = {
+        "logs": [
+            {"description": "added a feature that test", "type": "feature"},
+            {"description": "breaking changes: test", "type": "breaking"},
+        ],
+        "pr_number": "43524",
+    }
+LOG_FILE_3 = {
+    "logs": [
+            {"description": "added a feature that test", "type": "fix"},
+            {"description": "breaking changes: test", "type": "internal"},
+        ],
+        "pr_number": "43524",
+    }
+
+
+@pytest.fixture
+def changelog_mock():
+    return Changelog(pr_name="test", pr_number="12345")
+
+@pytest.fixture
+def changelog_folder_mock(tmpdir, mocker):
+    folder_path = Path(tmpdir / ".changelog")
+    folder_path.mkdir()
+    mocker.patch.object(changelog, "CHANGELOG_FOLDER", folder_path)
+    return folder_path
+
+
+@pytest.mark.parametrize(
+    "pr_name, expected_result", [("", False), ("test", False), ("v1.10.2", True)]
+)
+def test_is_release(pr_name: str, expected_result: bool):
+    """
+    Given:
+        - Changelog obj with some different `pr_name`
+    When:
+        - run the is_release method
+    Then:
+        - Ensure return True only if pr_name is in vX.X.X format
+    """
+    assert is_release(pr_name) == expected_result
+
+
+def test_is_log_folder_empty(changelog_folder_mock: Path):
+    """
+    Given:
+        - Changelog obj and a temporary path with a .changelog folder
+    When:
+        - run the is_log_folder_empty method
+    Then:
+        - Ensure return True only if there is a file in the .changelog folder
+    """
+    assert is_log_folder_empty()
+    (changelog_folder_mock / "12345.yml").write_text("test: test")
+    assert not is_log_folder_empty()
+
+
+@pytest.mark.parametrize("pr_name", ("", "12345"))
```

Comment:
Replace all `"12345"` with a constant `DUMMY_PR_NUMBER`.
This will make it clearer for future maintainers why `test_is_log_yml_exist` expects a file with this name